Sources/Parsing/Documentation.docc/Articles/Parsers/Bool.md

```diff
@@ -0,0 +1,66 @@
+# Bool
+
+A parser that consumes a Boolean value from the beginning of a string.
+ 
+This parser only recognizes the literal "true" and "false" characters:
+
+```swift
+// Parses "true":
+var input = "true Hello"[...]
+try Bool.parser().parse(&input)  // true
+input                            // " Hello"
+
+// Parses "false":
+input = "false Hello"[...]
+try Bool.parser().parse(&input)  // false
+input                            // " Hello"
+
+// Otherwise fails:
+input = "1 Hello"[...]
+try Bool.parser().parse(&input)
+
+// error: unexpected input
+//  --> input:1:1
+// 1 | 1 Hello
+//     ^ expected "true" or "false"
+```
+
+The `Bool.parser()` method is overloaded to work on a variety of string representations in order
+to be as efficient as possible, including `Substring`, `UTF8View`, and generally collections of 
+UTF-8 code units (see <doc:StringAbstractions> for more info).
```

Comment:
```suggestion
to be as efficient as possible, including `Substring`, `UTF8View`, and more general collections of 
UTF-8 code units (see <doc:StringAbstractions> for more info).
```

Reply:
(Take this or leave it. I think the current way reads OK on seeing it in the other docs, but it did read strange to me initially.)